owncloud/ios-app
# Fixed wrong gitattribute spelling

## Description
Fastlane build failed, because 
`git status --porcelain`
end with an error: 
` is not a valid attribute name: .gitattributes:1`


## Motivation and Context
Running fastlane build

## How Has This Been Tested?
- checkout master branch and run this terminal command in repo root:
`git status --porcelain`
this error should appear:
` is not a valid attribute name: .gitattributes:1`

- checkout branch `fix/gitattribute` and run this terminal command in repo root:
`git status --porcelain`
error above should not appear

## Types of changes


## Checklist:
